openedx/openedx-filters
# chore: deleting unused files

**Description:** This PR removes some artifacts from the cookie cutter that were never used.

**Merge deadline:** None. Just cleaning things up

**Testing instructions:**

1. check the green marks from the CI build. If it passes, then this files are not needed at all.

**Author concerns:** Could something in the CI break if this file is not present?
